starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,53 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = 0;
+}
+
+Chronometer.prototype.startClick = function () {
+    var self = this;
+
+    self.intervalId = setInterval (function(){
+                                            self.currentTime += 1; 
+                                            self.setTime();
+                                            }, 1000);
+
+};
+
+Chronometer.prototype.setMinutes = function () {
+    return Math.floor(this.currentTime/60); 
+};
+
+Chronometer.prototype.setSeconds = function () {
+    return  this.currentTime - Math.floor(this.currentTime/60)*60; 
```

Comment:
Your solution is fine, but there are easier ways to do it. First, you are repeating your code, so you can use your function instead:
```
return  this.currentTime - this.setMinutes() * 60;
```
However, there is a better approach. With your operations you are calculating the remainder, then go for it:
```
return  this.currentTime % 60;
```